docs/migration.md

```diff
@@ -57,6 +57,12 @@ removed and the new file name is `service-worker.js`.
 
 ### API changes
 
+#### `jupyterlite-core`
+
+- The Mathjax addon was removed from the `jupyterlite-core` package. As a consequence
```

Comment:
Could reference the upstream change, and maybe offer https://pypi.org/project/jupyterlab-mathjax2/ if someone wants the older behavior. Not sure if it can be configured...

Reply:
Also probably `MathJax`.

> because the RTD demo uses `jupyterlab-myst`

Given it breaks a fair amount of CSS, still, maybe it's not worth keeping that extension it if it hides (problems in) core, out-of-the-box features.

Reply:
Maybe. Although in this particular case the ignored 404s in https://github.com/jupyterlab/jupyterlab/pull/14726 should probably have been tracked in a separate issue (would have been easier to find the issue at least).